CGPs/0036/staging.json

```diff
@@ -0,0 +1,152 @@
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "",
+        "args": ["0x26412103865Be3F80152EA10E61E6294ed0e0DC9"],
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "",
+        "args": ["0x7Ba21D1f769Ae9feCF58dEfB5bD08905Bd3B707B"],
+        "value": "0"
+      },
+
+      // sent constitution parameters (for exchange) (same as the ones from governanceConstitution.js in monorepo)
+
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C", "", "0.9"], // setRegistry
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C", "", "0.8"], // setUpdateFrequency
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C", "", "0.8"], // setMinimumReports
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C", "", "0.8"], // setStableToken
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C", "", "0.8"], // setSpread
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C", "", "0.8"], // setReserveFraction
+        "value": "0"
+      },
+
+      // sent constitution parameters (for stableToken) (same as the ones from governanceConstitution.js in monorepo)
+
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0xcCA78ae41fe8F83eE7F37B0110eb6FAe3E9Ca0F0", "", "0.9"], // setRegistry
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0xcCA78ae41fe8F83eE7F37B0110eb6FAe3E9Ca0F0", "", "0.6"], // setInflationParameters
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0xcCA78ae41fe8F83eE7F37B0110eb6FAe3E9Ca0F0", "", "0.6"], // transfer
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0xcCA78ae41fe8F83eE7F37B0110eb6FAe3E9Ca0F0", "", "0.6"], // transferWithComment
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0xcCA78ae41fe8F83eE7F37B0110eb6FAe3E9Ca0F0", "", "0.6"], // approve
+        "value": "0"
+      },
+    ]
+    
```

Comment:
Is there a risk this proposal could get too big and may fail?

Reply:
The block gas limit is a hard limit but I don't think we're in danger of getting close to that